upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -525,9 +506,9 @@ There you can find links to upgrade notes for other versions too.
             - `Shopsys\FrameworkBundle\Model\Customer\UserData` to `Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserData`
             - `Shopsys\FrameworkBundle\Model\Customer\UserDataFactory` to `Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserDataFactory`
             - `Shopsys\FrameworkBundle\Model\Customer\UserDataFactoryInterface` to `Shopsys\FrameworkBundle\Model\Customer\User\CustomerUserDataFactoryInterface`
-            - `Shopsys\FrameworkBundle\Model\Customer\UserFacade` to `Shopsys\FrameworkBundle\Model\Customer\CustomerUserFacade`
```

Comment:
There are many instructions about changing user structure, I think it would be nice to create separate document about upgrading to new user structure.

Reply:
I do not see this as something important right now as added value would not be significant.